crypto/bn/bn_dh.c

```diff
@@ -104,6 +104,146 @@ static const BN_ULONG dh2048_256_q[] = {
     0x8CF83642A709A097ULL
 };
 
+/* Primes from RFC 7919 */
+static const BN_ULONG ffdhe2048_p[] = {
+    0xFFFFFFFFFFFFFFFFULL, 0x886B423861285C97ULL, 0xC6F34A26C1B2EFFAULL,
```

Comment:
It should be noted that standard says that type of constant will be automatically chosen to accommodate its value. Or in other words constants in question will be of right type without suffixes such as LL. On the other hand standard does specify LL, but some compilers might issue warning when it's assigned to long, or just say things about standard compliance, and others are not compliant and fail to recognize it. Last time I faced this dilemma (in blake2b) I reckoned that omitting suffix minimizes the chances of running into warning. In this case chances of running into problem are smaller than in blake2b, because this is compiled only on 64-bit platforms, while blake2b on both 32- and 64-bit platforms. So this means that this is side note, i.e. no action is expected, just a thing to keep in mind...

Reply:
On this (side) note about compiler choosing appropriate type, I do sometimes see a warning "this decimal constant is unsigned only in ISO C90" from asn1_time_test.c, but I did not dig into the details.